tutor/src/helpers/dates.js

```diff
@@ -33,9 +33,13 @@ export function findLatest(...dateThings) {
   return max(compact(map(flatten(dateThings), toDate)));
 }
 
-
 export function dateWithUnchangedTime(date, t) {
   const time = moment(t);
   return moment(date)
     .hour(time.hour()).minute(time.minute()).seconds(time.seconds()).millisecond(0);
 }
+
+export function guessISODateWithTimezone(isoDate) {
+  const timezone = moment.tz.guess(true);
```

Comment:
I don't think this is quite right... it should read the timezone from the course instead of using the browser's.

Reply:
I might be missing something @Dantemss . I am guessing the timezone of the user here so it reflects the date and time of the course dates in the user's timezone.